src/config.c

```diff
@@ -3058,6 +3058,7 @@ standardConfig static_configs[] = {
     createBoolConfig("replica-serve-stale-data", "slave-serve-stale-data", MODIFIABLE_CONFIG, server.repl_serve_stale_data, 1, NULL, NULL),
     createBoolConfig("replica-read-only", "slave-read-only", DEBUG_CONFIG | MODIFIABLE_CONFIG, server.repl_slave_ro, 1, NULL, NULL),
     createBoolConfig("replica-ignore-maxmemory", "slave-ignore-maxmemory", MODIFIABLE_CONFIG, server.repl_slave_ignore_maxmemory, 1, NULL, NULL),
+    createBoolConfig("replica-redirect-read-write", NULL, MODIFIABLE_CONFIG, server.repl_replica_redirect_rw, 0, NULL, NULL),
```

Comment:
Is `replica-redirect` enough? It can be implicitly understood that it is only for read and write commands, just like for cluster redirects.

Reply:
I actually am not sure it's clear enough! We've had folks ask if they could disable redirects for cluster mode (for mostly bad reasons), so they might think they could turn this off for cluster mode.

Reply:
So you want to add something like 'standalone' to the name to make it clear it's not a cluster config?

Reply:
I guess in my ideal world this config would also apply on cluster mode enabled as well, and it would be a cluster agnostic config. At that point maybe your naming would be sufficient, `replica-enable-redirects` maybe?

Reply:
This is already the behaviour in cluster mode. Would the config have different defaults in cluster vs standalone mode. Btw disabling it in cluster mode is "for mostly bad reasons" you said above so I thought you don't want to allow that...

Reply:
i agree it would be nice to have it cluster agnostic, and that then we have a problem with default.
unless we wanna make it default to `yes` in standalone too.

another bad option is to make it an enum config with [yes,no,cluster-only], the last one being the default.

Reply:
if we want this, I prefer split into two configs, like `replica-redirect` by default `no` and `cluster-replica-redirect` by default `yes`.